autoware_adapi_v1_msgs/perception/msg/Shape.msg

```diff
@@ -0,0 +1,7 @@
+uint8 BOUNDING_BOX=0
```

Comment:
Could you avoid using zero or assign `UNKNOWN` to distinguish from uninitialized value?

Reply:
you mean 0 is used for `UNKNOWN` purpose?
I can change it

Reply:
> you mean 0 is used for UNKNOWN purpose?

Yes.

> This message is refer from here

Constants do not have to be the same as the internal messages, nor should we expect them to be. In the API adapter, even if the values are the same, they need to be converted using a switch statement, etc., for future changes.

Reply:
changed at [bbbfc45](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/25/commits/bbbfc45df9dc751fa5513bdc421791cd022f60fc)